app/emu_svc.py

```diff
@@ -149,7 +169,7 @@ async def _save_ability(self, ab):
             id=ab.pop('id', str(uuid.uuid4())),
             name=ab.pop('name', ''),
             description=ab.pop('description', ''),
-            tactic=ab.pop('tactic', None),
+            tactic='-'.join(ab.pop('tactic', None).lower().split(' ')),
```

Comment:
Based on the behavior of this line. I think replacing it with empty string as default is a safer option. Otherwise it will try to do `None.lower()` triggering an AttributeError.

```suggestion
            tactic='-'.join(ab.pop('tactic', '').lower().split(' ')),
```